operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -645,36 +673,36 @@ private Storage buildKafkaStorage(ManagedKafka managedKafka, Kafka current) {
     /**
      * Get per broker value
      */
-    private long getPerBrokerBytes(ManagedKafka managedKafka, Function<ManagedKafka, Quantity> capacityValue, Supplier<String> defaultValue) {
+    private long getPerBrokerBytes(ManagedKafka managedKafka, Kafka current, Function<ManagedKafka, Quantity> capacityValue, Supplier<String> defaultValue) {
```

Comment:
Why do we pass a Function here if we just call it immediately? Why not just pass the value directly? Same with the `Supplier`.

Reply:
We certainly can just pass the value.  In a different version of the method the default was optional, it can be refactored again to work that way.